setup.py

```diff
@@ -37,12 +44,25 @@ def run(self):
 
         with tempfile.NamedTemporaryFile() as download_file:
             log.info(f"Downloading UI distribution to temporary file: {download_file.name}")
-            urllib.request.urlretrieve(self.UI_DOWNLOAD_URL, filename=download_file.name)
+
+            if (self.ALTERNATE_UI_DOWNLOAD_URL):
```

Comment:
Parenthesis are redundant here

Reply:
woops. This is what I get from constantly switching between javascript and python